src/modules/layout/components/Header.js

```diff
@@ -58,15 +58,11 @@ const MaterialHeaderButtons = props => {
 }
 
 type PropsType = {|
+  navigation: NavigationScreenProp<*>,
```

Comment:
Can you try there to replace:
https://github.com/Integreat/integreat-react-native-app/blob/a0fb69e33ae78d8be23242b57b47a4f4326d1f10/src/modules/layout/components/Header.js#L85

with `this.props.navigation`? Would fix quite a few flow errors in this file. Also it would be a validation if Header really has the navigation prop.

Reply:
Done!